src/backend/joanie/settings.py

```diff
@@ -437,9 +438,15 @@ class Base(Configuration):
         environ_name="JOANIE_WITHDRAWAL_PERIOD_DAYS",
         environ_prefix=None,
     )
+    # Email for installment payment
+    # Add here the dashboard link of orders
+    JOANIE_DASHBOARD_ORDER_LINK = values.Value(
+        "/dashboard/courses/orders/:orderId/",
+        environ_name="JOANIE_DASHBOARD_ORDER_LINK",
+        environ_prefix=None,
+    )
```

Comment:
I just realized that's should be a full url because currently, the hostname is the joanie domain name.

IMO, to make things right, in the `Base` Configuration, you should let default and for dev purpose you could set the value into `env.d/development/common`